### PR TITLE
add python exercises

### PR DESCRIPTION
**Pull request by FarhadJahangiri**

### CODE REVIEW

File: python/min.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mylist=[1, 2, 3]
+print(min(mylist))
```
Comment (mbovel) [inline python/min.py:2]: The goal was to implement your own `min` function, but that also works 😉

File: python/sum.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+numbers = [1,2,3]
+sum = sum(numbers)
```
Comment (mbovel) [inline python/sum.py:2]: Same comment here.

**Comment by mbovel** [comment]
Please add @msom as a contributor to your repository and as a reviewer to this PR.

File: python/random_password.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import string
+length = random.randint(6,10)
+password = random.choice(string.ascii_letters)
+for i in range(length):
+    a = random.choice(string.ascii_letters)
+    n = random.choice(string.digits)
+    password += a
+    password += n
+
+shufPass = ''.join(random.sample(password, len(password)))
```
Comment (msom) [inline python/random_password.py:11]: you could also use random.shuffle which will be easier to understand

**Comment by msom** [comment]
Looks good!

**Pull request merged**